phoenix-core/src/main/java/org/apache/phoenix/parse/ParseNodeFactory.java

```diff
@@ -127,6 +129,7 @@ private static void addBuiltInFunction(Class<? extends FunctionExpression> f) th
         }
         int nArgs = d.args().length;
         BuiltInFunctionInfo value = new BuiltInFunctionInfo(f, d);
+        SINGLE_SIGNATURE_BUILT_IN_FUNCTION_MAP.put(value.getName(), value);
```

Comment:
We'll have to adjust the map initialization logic here according to the new annotation. Avoid "ABSTRACT" for "SINGLE_SIGNATURE_BUILT_IN_FUNCTION_MAP" and "DERIVED" for "BUILT_IN_FUNCTION_MAP".